diff --git a/node_modules/vite/dist/node/chunks/dep-cc49d7be.js b/node_modules/vite/dist/node/chunks/dep-cc49d7be.js
index 2b96b60..c03ecfd 100644
--- a/node_modules/vite/dist/node/chunks/dep-cc49d7be.js
+++ b/node_modules/vite/dist/node/chunks/dep-cc49d7be.js
@@ -33793,8 +33793,7 @@ function getName(id) {
   if (name !== 'index') {
     return name;
   }
-  const segments = path$A.dirname(id).split(path$A.sep);
-  return makeLegalIdentifier(segments[segments.length - 1]);
+  return makeLegalIdentifier(path$A.basename(path$A.dirname(id)));
 }
 
 function normalizePathSlashes(path) {
